build.rs

```diff
@@ -7,9 +9,11 @@ const CONFIG: &str = include_str!("configs/env.json");
 #[derive(serde::Deserialize)]
```

Comment:
What about a more general (os,arch) matrix:

```rust
#[derive(Debug, Eq, PartialEq, Hash)]
enum Os {
   Linux,
   MacOS
}

impl TryInto<Os> for &str {
   fn try_into(&str) -> Error<Os, ()> {
      ...
   }
}

#[derive(Debug, Eq, PartialEq, Hash)]
enum Arch {
   Aarch64,
   Amd64
}



struct Artifacts {
   url: String,
   sha256_sums: Vec<String>
}

struct Config {
   dists: HashMap<(Os, Arch), Artifacts>
}
```

Reply:
e.g. I suspect that with Arm64, they could also support (Aarch64, Linux) pretty easily.

Reply:
Done, will explore (aarch64, linux) option later.